openghg/store/_populate.py

```diff
@@ -81,12 +84,19 @@ def add_noaa_obspack(
     files_with_errors = []
     # Find relevant details for each file and call parse_noaa() function
     processed_summary: Dict[str, Dict] = {}
+
+    # Object to generate progress bar
+    progress = Progress()
+    progress.start()
+    task1 = progress.add_task("Downloading...", total=len(files))
+
```

Comment:
Could we use `track` here?

Reply:
Turns out it is possible because of using RichHandler